modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/PythonClientCodegen.java

```diff
@@ -102,7 +102,7 @@ public PythonClientCodegen() {
                     "and", "del", "from", "not", "while", "as", "elif", "global", "or", "with",
                     "assert", "else", "if", "pass", "yield", "break", "except", "import",
                     "print", "class", "exec", "in", "raise", "continue", "finally", "is",
-                    "return", "def", "for", "lambda", "try", "self", "None"));
+                    "return", "def", "for", "lambda", "try", "self", "nonlocal", "None", "True", "False"));
```

Comment:
https://github.com/swagger-api/swagger-codegen/issues/7955#issuecomment-384636747

Can you please add `type` and `id` as well all other built-in functions? These keywords are not prohibited as the reserved words, but it is generally good idea not to use them in code, as they are masking the built-in functions. Since there are already some keywords in the list that are not reserved (including `self`, which is nothing but common naming best practice to reference to the instance), I suggest expanding the list of reserved keywords to include all the [built-in functions](https://docs.python.org/3/library/functions.html).